package.json

```diff
@@ -69,6 +69,7 @@
         "onCommand:aws.ssmDocument.publishDocument",
         "onCommand:aws.ssmDocument.updateDocumentVersion",
         "onLanguage:javascript",
+        "onLanguage:java",
```

Comment:
Now that 'onStartupFinished' is an activation event, do we need to keep adding commands to this list?

Reply:
I think this might prioritize the extension loading a bit faster. I don't think it hurts to keep it around.